### PR TITLE
Fix request insert

### PR DESCRIPTION
**Pull request by dzflack**
- Fix minor formatting and add files to gitignore.
- Add *UNIQUE* constraint to URI column.
- Insert initial request URIs into database using a transaction.

### CODE REVIEW

File: libgetgood/updater.go [hunk 1]
```diff
@@ -48,7 +48,9 @@ func (updater *Updater) work() {
 	for running {
 		select {
 		case r := <-updater.requestChan:
+			log.Printf("Adding URLs to databse\n")
 			err := updater.addURLs(r.Url)
+			log.Printf("URLs Added to database\n")
```
Comment (dpindur) [inline libgetgood/updater.go]: I think we can drop these two log statements as they were mainly for debugging the issue.

If we want to include them later I think we should put them in the addURLs function as it's called elsewhere as well.

File: libgetgood/database.go [hunk 2]
```diff
@@ -34,24 +36,29 @@ func (conn *DBConn) Clear() error {
...
-	default:
-		return true, nil
+	insertURI, err := conn.db.Prepare("INSERT OR IGNORE INTO requests (status, uri) VALUES (?, ?)")
+	if err != nil {
+		return err
+	}
+
+	tx, err := conn.db.Begin()
+	if err != nil {
+		log.Printf("Failed to begin transaction")
```
Comment (dpindur) [inline libgetgood/database.go]: We can drop this as the error bubbles up and is printed elsewhere

**Review by dpindur — CHANGES_REQUESTED** [review]
Looks great! Just got to clean up some logging and then we're all good to go! 👌

**Comment by dzflack** [comment]
Requested changes made in d729cae7a8507055796c36101eaa2e946debd2b1.

**Review by dpindur — APPROVED** [review]

**Pull request merged**